docs/dev/table/connectors/formats/avro.md

```diff
@@ -97,6 +97,13 @@ Format Options
       <td>String</td>
       <td>Specify what format to use, here should be 'avro'.</td>
     </tr>
+    <tr>
+      <td><h5>avro.codec</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>For <a href="{{ site.baseurl }}/dev/table/connectors/filesystem.html">Filesystem</a> only, the compression codec for avro. Default no compression. Support deflate, snappy, bzip2, xz.</td>
+    </tr>
```

Comment:
No compression as default. The valid enumerations are: `deflate`, `snappy`, `bzip2`, `xz`.

Reply:
Let's use `{% link dev/table/connectors/filesystem.md %}` as the link which is suggested in the dev mailing list recentlly.